mig-agent/config.go

```diff
@@ -167,9 +183,30 @@ type globals struct {
 
 	// Add the private client key below.
 	agentKey []byte
+
+	// let you target specific agents
+	tags map[string]string
+}
+
+func globalTagsMap() map[string]string {
+	tags := make(map[string]string)
+	inspectTags := reflect.TypeOf(&TAGS).Elem()
+
+	for tag, val := range structs.Map(TAGS) {
+		field, ok := inspectTags.FieldByName(tag)
+
+		if ok {
+			_, tag = stringPair(string(field.Tag))
+			tag = strings.Replace(tag, `"`, "", -1)
+		}
+		tags[tag] = fmt.Sprintf("%v", val)
+	}
+	return tags
```

Comment:
Ok, changing the type will definitely make things easier, I'll be able to remove some of my workaround.